robot/resources/files/container_policy.json

```diff
@@ -0,0 +1,3 @@
+{
+    "rep-1": "REP 1"
```

Comment:
Maybe add the number of REP greater than 1?

Reply:
it should be added when it is needed, for current pr, there is no need to add more than 1